docs/_documentations/workingwithtemplates.md

```diff
@@ -31,16 +31,25 @@ GitHub repository. Three examples are included in Codewind for your reference:
 * Appsody Stacks to develop applications with sharable technology stacks. 
 
 Use the **Template Source Manager** to perform the following actions:
-1. To add a new template source to the table, click **Add New**. For more information, see [Adding your own template sources to Codewind](#adding-your-own-template-sources-to-codewind).
-2. To remove non-default template sources, click the trash icon. 
-3. Toggle the **Enabled** slide to **On** so template source templates appear in the **Create Project** wizard. 
+
+1. For more information, see [Adding your own template sources to Codewind](#adding-your-own-template-sources-to-codewind).
+
+2. (**VSCode**) To remove non-default template sources, click the trash icon.
```

Comment:
Change (**VSCode**) to:   **VS Code**: 
this to match other occurrences of the same in other topics.